algebird-test/src/test/scala/com/twitter/algebird/QTreeTest.scala

```diff
@@ -69,6 +69,15 @@ class QTreeTest extends WordSpec with Matchers {
     list.filter{ _ >= from }.filter{ _ < to }.sum
```

Comment:
we don't need the outer `{ }`, and this could be `trueRangeSum[T: Monoid](l: TraversableOnce[T], int: Interval[T]): T` to be more general